lib/streamlit/user_info.py

```diff
@@ -28,7 +28,33 @@ def _get_user_info() -> UserInfo:
 
 
 class UserInfoProxy(Mapping[str, Optional[str]]):
-    """A dict like proxy object for accessing information about current user."""
+    """
+    A read-only, dict-like object for accessing information about current user.
+
+    ``st.experimental_user`` is dependant on the host platform running the
+    Streamlit app. If the host platform has not configured the function, it
+    will behave as it does in a locally running app.
+
+    Properties can by accessed via key or attribute notation. For example,
+    ``st.experimental_user["email"]`` or ``st.experimental_user.email``.
+
+    Parameters
+    ----------
```

Comment:
I think this section should be an `Attributes` section instead of a parameters section

Reply:
There is a limitation of the docstring parser where it can't itemize "Attributes" like it can "Parameters" so I did a workaround. I intend to rework the parser later, but that will take a little work. If it's very objectionable, I can try and see if I can use a string replacement to change "Attributes" to "Parameters" internally so that the workaround isn't as apparent. LMK. 😁

Reply:
Ah, in that case I think this should be good to go as-is. I think we'll want a note in the docstring itself mentioning why this section seems to be incorrectly labeled, though

Reply:
Would it be alright to add a comment above the function? When the docstring renders on the docs website, it will correctly show as "Attributes" so I wouldn't want to add text inside the docstring about it. Here's how the docstring appears as they are in this commit: https://docs.streamlit.io/library/api-reference/personalization/st.experimental_user

Reply:
Ah, yeah above the function should be fine. I originally thought that it'd be better to have it in the docstring itself since it'll still show up as "Parameters" when the `UserInfoProxy` class is printed via `st.help` (which I think automatically happens with `st.write` as well. I'd imagine that most people are using the docs page, though, rather than writing the class to an app and reading the documentation there, so we should make it less awkward in the more common case